### ISSUE TITLE
Redirect from /actions/leaving-money-to-charity-in-your-will

### ISSUE THREAD

**Issue by taymonbeal** [rtcharity/eahub.org#652]
Needs a page on the Resource Trail.

**Comment by taymonbeal** [comment]
Blocked on resources.eahub.org#374.

**Comment by victor-yunenko** [comment]
Closing in favor of #999
